internal/service/ch/ch.go

```diff
@@ -90,9 +90,17 @@ func (s *Service) GetLatestSignals(ctx context.Context, latestArgs *model.Latest
 // GetAggregatedSignals returns a slice of aggregated signals based on the provided arguments from the ClickHouse database.
 // The signals are sorted by timestamp in ascending order.
 // The timestamp on each signal is for the start of the interval.
-func (s *Service) GetAggregatedSignals(ctx context.Context, aggArgs *model.AggregatedSignalArgs) ([]*vss.Signal, error) {
-	stmt, args := getAggQuery(aggArgs)
-	signals, err := s.getSignals(ctx, stmt, args)
+func (s *Service) GetAggregatedSignals(ctx context.Context, aggArgs *model.AggregatedSignalArgs) ([]*model.AggSignal, error) {
+	if len(aggArgs.FloatArgs) == 0 && len(aggArgs.StringArgs) == 0 {
```

Comment:
None of these temporary table things work well when the temporary table is empty, so short circuit here.